config/labels/analyzers/clang-tidy.json

```diff
@@ -5645,6 +5688,10 @@
       "doc_url:https://clang.llvm.org/extra/clang-tidy/checks/readability-container-size-empty.html",
       "severity:STYLE"
     ],
+    "readability-container-data-pointer": [
+      "doc_url:https://clang.llvm.org/extra/clang-tidy/checks/readability-data-pointer.html",
```

Comment:
I get 404 error when I visit this url. In this case it should be removed or please fix the url.